### ISSUE TITLE
Table headings missing a text alternative

### ISSUE THREAD

**Issue by muffinresearch** [mozilla/extension-workshop#440]
See https://extensionworkshop-dev.allizom.org/documentation/develop/firefox-workflow-overview/

Each `th` contains a png with text (in the image) that has no alt attribute set:

<img width="673" alt="Firefox_workflow_overview___Extension_Workshop" src="https://user-images.githubusercontent.com/1514/65448409-9d474180-de30-11e9-8dab-a1e966902132.png">

<img width="855" alt="Firefox_workflow_overview___Extension_Workshop" src="https://user-images.githubusercontent.com/1514/65448462-bea82d80-de30-11e9-9a34-d3b1bdb63e70.png">

**Comment by caitmuenster** [comment]
For the time being, I'd be ok with removing the images and having the headers be bolded in the larger font size.

@championshuttler, do you want to take this on?
